integrations/server-express/README.md

```diff
@@ -1,37 +1,20 @@
----
-title: 'ExpressJS Server Integration'
-excerpt: 'Host Jovo apps on your own server using ExpressJS.'
----
+# Jovo Express Integration
```

Comment:
Use ExpressJS to be more consistent?

Reply:
I ended up changing "ExpressJS" to "Express" everywhere because this seems to be the preferred way how they call themselves in their docs